components/atoms/ActionButton.js

```diff
@@ -7,6 +7,7 @@ import Link from 'next/link'
 export function ActionButton({
   href,
   className,
+  extendedClass,
```

Comment:
Since we are not using it, we can remove the extendedClass in both places (line 10 and line 87)

Reply:
Oops, yes, I forgot those